app/src/androidTest/java/com/github/se/cyrcle/ui/report/ParkingReportScreenTest.kt

```diff
@@ -0,0 +1,134 @@
+class ParkingReportScreenTest {
+
+  @get:Rule val composeTestRule = createComposeRule()
+
+  private lateinit var parkingRepository: ParkingRepository
+  private lateinit var userRepository: UserRepository
+  private lateinit var reviewRepository: ReviewRepository
+  private lateinit var reportedObjectRepository: ReportedObjectRepository
+
+  private lateinit var userViewModel: UserViewModel
+  private lateinit var parkingViewModel: ParkingViewModel
+  private lateinit var reviewViewModel: ReviewViewModel
+
+  private lateinit var navigationActions: NavigationActions
+
+  @Before
+  fun setUp() {
+    navigationActions = mock(NavigationActions::class.java)
+
+    parkingRepository = MockParkingRepository()
+    userRepository = MockUserRepository()
+    reviewRepository = MockReviewRepository()
+    reportedObjectRepository = MockReportedObjectRepository()
+
+    parkingViewModel =
+        ParkingViewModel(MockImageRepository(), parkingRepository, reportedObjectRepository)
+    userViewModel =
+        UserViewModel(
+            userRepository,
+            parkingRepository,
+            MockImageRepository(),
+            MockAuthenticationRepository())
+    reviewViewModel = ReviewViewModel(reviewRepository, reportedObjectRepository)
+
+    parkingViewModel.addParking(TestInstancesParking.parking2)
+    parkingViewModel.addParking(TestInstancesParking.parking3)
+
+    `when`(navigationActions.currentRoute()).thenReturn(Screen.PARKING_DETAILS)
+  }
+
+  @Test
+  fun parkingReportScreen_displaysCorrectly() {
+    composeTestRule.setContent {
+      ParkingReportScreen(
+          navigationActions = navigationActions,
+          userViewModel = userViewModel,
+          parkingViewModel = parkingViewModel)
+    }
+
+    // Assert that the report title is displayed
+    composeTestRule.onNodeWithTag("ReportTitleText").assertIsDisplayed()
+
+    // Assert that the bullet points section is displayed
+    composeTestRule.onNodeWithTag("ReportBulletPoints").assertExists()
+
+    // Assert that the dropdown for reason selection is displayed
+    composeTestRule.onNodeWithTag("ReasonDropDown").assertExists()
+
+    // Assert that the input field for additional details is displayed
+    composeTestRule.onNodeWithTag("ReportDetailsInput").assertExists()
+
+    // Assert that the submit button is displayed
+    composeTestRule.onNodeWithTag("SubmitButton").assertExists()
+  }
+
+  @OptIn(ExperimentalTestApi::class)
+  @Test
+  fun submitButton_displaysDialog_whenClicked() {
+    composeTestRule.setContent {
+      ParkingReportScreen(
+          navigationActions = navigationActions,
+          userViewModel = userViewModel,
+          parkingViewModel = parkingViewModel)
+    }
+
+    // Click on the submit button
+    composeTestRule.onNodeWithTag("SubmitButton").performClick()
+    // Assert that the dialog exists
+    composeTestRule.onNodeWithTag("ReportScreenAlertDialog").assertExists()
+  }
+
+  @Test
+  fun detailsInput_acceptsTextInput() {
```

Comment:
That's a minor issue but the underscores in the name of the tests are really weird because I dont see it anywhere else